examples/inverse/plot_extract_functional_label.py

```diff
@@ -0,0 +1,82 @@
+"""
+========================================
+Extracting a functional label from a STC
+========================================
+
+We first apply a dSPM inverse operator to get source activations.
+Then we make a functional label from the contiguous patch of activity within
+33% of the peak activation level, at the peak activation time.
+"""
+# Author: Eric Larson <larson.eric.d@gmail.com>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+import mne
+from mne.datasets import sample
+from mne.minimum_norm import read_inverse_operator, apply_inverse
+
+print(__doc__)
+
+data_path = sample.data_path()
+fname_inv = data_path + '/MEG/sample/sample_audvis-meg-oct-6-meg-inv.fif'
+fname_evoked = data_path + '/MEG/sample/sample_audvis-ave.fif'
+subjects_dir = data_path + '/subjects'
+perc_thresh = 0.33
+
+snr = 3.0
+lambda2 = 1.0 / snr ** 2
+method = "dSPM"  # use dSPM method (could also be MNE or sLORETA)
+
+# Load data
+evoked = mne.read_evokeds(fname_evoked, condition=0, baseline=(None, 0))
+inverse_operator = read_inverse_operator(fname_inv)
+src = inverse_operator['src']
+
+# Compute inverse solution
+stc = apply_inverse(evoked, inverse_operator, lambda2, method)
+
+# Figure out the index of the maximum, and plot it
+max_vert_idx, max_t_idx = np.unravel_index(np.argmax(stc.data), stc.data.shape)
```

Comment:
I think there is a method for this:

https://www.martinos.org/mne/stable/generated/mne.SourceEstimate.html#mne.SourceEstimate.get_peak